### ISSUE TITLE
Tensorflow warning and information shown in console

### ISSUE THREAD

**Issue by stefanievonk** [amaiya/ktrain#160]
When loading a predictor using ktrain.load_predictor('file_path'), multiple Tensorflow warnings are pushed to the console, even when os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' is used.

`
2020-06-05 13:23:09.059775: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'nvcuda.dll'; dlerror: nvcuda.dll not found
2020-06-05 13:23:09.060227: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-06-05 13:23:09.061696: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: NLAMS08325
2020-06-05 13:23:09.062113: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: NLAMS08325
2020-06-05 13:23:09.062703: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
`

Running code in PyCharm on a Windows machine.

**Comment by amaiya** [comment]
Everything is working correctly both on Google Colab and locally on an Ubuntu machine.  Windows  users of *ktrain* have not reported problems like this.    It sounds like your GPU setup is incorrect in some way.  

Are you using TensorFlow 2.1.0?  Are you sure this error is occurring only when `load_predictor` is called and not during training?  What does `nvidia-smi` show during training? 
 Also, what kind of model are you using? Text classification with `transformer` model?  NER? Image classification?

**Comment by stefanievonk** [comment]
I am not using a GPU, that is probably also why it is giving those warnings. I am not really interested in how to fix the issue, but rather how to mute the warnings to the console.

**Comment by amaiya** [comment]
Are you using TensorFlow 2.1.0?

**Comment by stefanievonk** [comment]
Yes, that specific TF version indeed and ktrain version 0.16.0

**Comment by amaiya** [comment]
TF 2.1.0 emits that warning when running on a machine with only CPU (see [here](https://stackoverflow.com/questions/59823283/could-not-load-dynamic-library-cudart64-101-dll-on-tensorflow-cpu-only-install)).  The warnings should be able to be suppressed.

Internally, *ktrain* aggressively suppresses such warnings by default using `TF_CPP_MIN_LOG_LEVEL=3` .  (In fact, the warnings must be explicitly enabled to see them by setting SUPPRESS_TF_WARNINGS to '0'.)    Why it is still showing up for you is not clear.  Perhaps it is a bug in TF 2.1.0.

I can look into it further, but you'll need to provide more details like which model you're using.

**Comment by stefanievonk** [comment]
That is also why I got confused that I was still seeing warning messages. I am using an own trained NER model. Let me see if I can reproduce it also with a pretrained model.

**Comment by amaiya** [comment]
On Windows, I used ktrain in an Ubuntu VM using VirtualBox (instead of Windows directly).  I ran the example using a CPU only and did not see any warnings.   My assumption is that this issue is something related to Windows and TensorFlow 2 (and not *ktrain*) that is causing `TF_CPP_MIN_LOG_LEVEL` to not be followed.  We will be upgrading *ktrain* to TensorFlow 2.2.0 in near future at which time maybe this will go away.
